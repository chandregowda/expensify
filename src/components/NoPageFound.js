import React from 'react';
import { Link } from 'react-router-dom';

const NoPageFound = () => (
	<p>
		404! Route not found - <Link to="/">Go Home</Link>
	</p>
);

export default NoPageFound;
