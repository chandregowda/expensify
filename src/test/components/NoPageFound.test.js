import React from 'react';
import { shallow } from 'enzyme';
import NoPageFound from '../../components/NoPageFound';

test('should display 404 No Page found error', () => {
	const wrapper = shallow(<NoPageFound />);
	expect(wrapper).toMatchSnapshot();
});
