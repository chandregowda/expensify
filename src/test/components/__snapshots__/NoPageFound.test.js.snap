// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display 404 No Page found error 1`] = `
<p>
  404! Route not found - 
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</p>
`;
